styles/blocks/f-info-card.css

```diff
@@ -0,0 +1,115 @@
+/*
+A numbered block with heading and separator. Below the separator, is text and image. 
+As seen on "How does it work".
+
+## f-info-card
+```
+<div class="kalei-background-white"> <!-- only here for the style guide - should not be used on the website -->
+<!-- this should be used on the website -->
+<div class="f-info-card">
+	<section class="f-row f-row_6">
+		<div class="f-row__cell-3 f-row__cell_push_1 f-row__cell-mobile">
+			<p class="f-info-card f-info-card_number">1</p>
```

Comment:
only one BEM block/element per html tag + modifier.
e.g. `<p class="f-info-card_number f-info-card__number_MODIFIER">1</p>`
and **not** `<p class="f-info-card f-info-card_number">1</p>`